### ISSUE TITLE
use logging api

### ISSUE THREAD

**Issue by PeterNSteinmetz** [jniediek/combinato#79]
Another change which would make combinato function better as a library is to use logging statements rather than print statements.

The same console output as presently can be achieved for the command line commands, if desired, by configuring a logging format to just give the message.

**Comment by jniediek** [comment]
I completely agree, it's a great and natural idea I think. Would you be willing to create a pull request?
Cheers,
Johannes

**Comment by jniediek** [comment]
Implemented by @ZhidongZhang96
